### ISSUE TITLE
account reload

### ISSUE THREAD

**Issue by benjyz** [CryptoverseCC/ethmail.cc#7]
I tried changing accounts while in web.ethmail and it doesn't update the account

**Comment by gkapkowski** [comment]
Yes, thanks for reporting. The web client doesn't yet observe wallet changes.
